monitoring/explores/mlops_flow_cost.explore.lkml

```diff
@@ -0,0 +1,6 @@
+include: "../views/mlops_flow_cost.view.lkml"
+
+explore: mlops_flow_cost {
+  label: "MLOPs Aggregated Flow Cost Across Runs"
+
```

Comment:
nit: extra space
```suggestion
```

Reply:
Ah, I think I thought this was a convention also due to the particular examples I looked at.